Exabyte_TeamRoom3.md

```diff
@@ -14,8 +14,16 @@
   - [ ] Other
         - if other specify: ...
 * Experimental Data: 
-  * Describe the classes and types of materials covered. 
-    *  ....
+  * Calculate the properties of the material (DFT)
```

Comment:
thanks @GR12345678  for the commit, there are some useful things. for this one here though, DFT is modelling, so it should not be under experimental data but under modelling. perhaps these can be added under DFT better.